Plugin~/WebRTCPlugin/GraphicsDevice/D3D11/D3D11GraphicsDevice.h

```diff
@@ -10,14 +10,13 @@ class D3D11GraphicsDevice : public IGraphicsDevice{
 public:
     D3D11GraphicsDevice(ID3D11Device* nativeDevice);
     virtual ~D3D11GraphicsDevice();
-    virtual bool InitV();
-    virtual void ShutdownV();
-    inline virtual void* GetEncodeDevicePtrV();
-    virtual ITexture2D* CreateDefaultTextureV(uint32_t w, uint32_t h);
-    virtual ITexture2D* CreateDefaultTextureFromNativeV(uint32_t w, uint32_t h, void* nativeTexturePtr);
-    virtual bool CopyResourceV(ITexture2D* dest, ITexture2D* src);
-    virtual bool CopyResourceFromNativeV(ITexture2D* dest, void* nativeTexturePtr);
-    inline virtual GraphicsDeviceType GetDeviceType();
+    virtual bool InitV() override;
+    virtual void ShutdownV() override;
+    inline virtual void* GetEncodeDevicePtrV() override;
+    virtual ITexture2D* CreateDefaultTextureV(uint32_t w, uint32_t h) override;
+    virtual bool CopyResourceV(ITexture2D* dest, ITexture2D* src) override;
+    virtual bool CopyResourceFromNativeV(ITexture2D* dest, void* nativeTexturePtr) override;
+    inline virtual GraphicsDeviceType GetDeviceType() override;
```

Comment:
This looks like a const method.
```cpp
inline virtual GraphicsDeviceType GetDeviceType() const override;
```

Reply:
Yeah, it is a const method.
It doesn't return `const GraphicsDeviceType` since `GraphicsDeviceType` is just an enum, and therefore writing `const GraphicsDevice` is not meaningful.

Reply:
Ah, oops, never mind what I said. Fixing it.